Tests/Droid/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="53" android:versionName="1.9.1-SNAPSHOT" package="com.contoso.contoso_forms_test">
-    <uses-sdk android:minSdkVersion="15" />
-    <application android:label="Contoso.Forms.Test">
-    </application>
-</manifest>
+	<uses-sdk android:minSdkVersion="16" />
```

Comment:
We modify via UI, formatting is weird but trying to reformat would be canceled by using the UI of Visual Studio.